src/main/resources/templates/index.html

```diff
@@ -19,7 +19,7 @@
 </main>
 
 <footer>
-  <span>&copy; {cdi:web.year} mvnpm.io | v{cdi:web.version}</span>
```

Comment:
@phillip-kruger why do we need the api for the version (and not the year)? Qute should work, I think there is something wrong in Qute or Config, but we shouldn't need an api here?

Reply:
cc @mkouba

Reply:
I don't think the config resolve in build time

Reply:
there are some template stuff happening at runtime, I think we are missing something, let's see what Martin says

Reply:
Since the `WebData` is `@ApplicationScoped` you should never access the fields of the bean directly. Because you read the state of the [client proxy](https://quarkus.io/guides/cdi#client_proxies) and not the bean itself.

You should either switch to `@Singleton` or use getters (methods in general). The difference between `@ApplicationScoped`  and `@Singleton`   is explained [here](https://quarkus.io/guides/cdi#applicationscoped-and-singleton-look-very-similar-which-one-should-i-choose-for-my-quarkus-application).

HTH

Reply:
Thanks for the clarification @mkouba :) @phillip-kruger will you do the fix or should I?

Reply:
Thanks Martin. Andy - go for it!